library/src/main/java/com/getstream/sdk/chat/utils/Utils.java

```diff
@@ -117,20 +115,37 @@ public static void showMessage(Context context, String message) {
         toast.show();
     }
 
-    public static void showSoftKeyboard(Activity activity) {
-        InputMethodManager inputMethodManager = (InputMethodManager) activity.getSystemService(Context.INPUT_METHOD_SERVICE);
-        if (!inputMethodManager.isAcceptingText())
-            inputMethodManager.toggleSoftInput(InputMethodManager.SHOW_FORCED, 0);
+    public static void showSoftKeyboard(@NonNull Context context) {
+        InputMethodManager imm = (InputMethodManager) context.getSystemService(Context.INPUT_METHOD_SERVICE);
+        if (!imm.isAcceptingText())
+            imm.toggleSoftInput(InputMethodManager.SHOW_FORCED, 0);
     }
 
-    public static void hideSoftKeyboard(Activity activity) {
-        InputMethodManager imm = (InputMethodManager) activity.getSystemService(Activity.INPUT_METHOD_SERVICE);
-        //Find the currently focused view, so we can grab the correct window token from it.
-        View view = activity.getCurrentFocus();
-        //If no view currently has focus, create a new one, just so we can grab a window token from it
+    public static void hideSoftKeyboard(@NonNull Context context) {
```

Comment:
This method could call to `public static void hideSoftKeyboard(@NonNull View view)` method, and reuse the same logic

Reply:
True, I was doing this the other way around with the _show_ methods already, not sure why I didn't do it here. Updated the PR 😄